File: Interpreter/src/Interface/AllStepsCommand.java
package Interface;

import ExceptionHandling.MyException;
import controller.Controller;

import java.util.Scanner;

public class AllStepsCommand extends Command{
    private Controller ctr;
    public AllStepsCommand(String key, String desc,Controller ctr){
        super(key, desc);
        this.ctr=ctr;
    }
    @Override
    public void execute() {
        Scanner scan = new Scanner(System.in);
        System.out.println("Index of Program :");
        String index = scan.nextLine();
        try {
            ctr.AllSteps(Integer.parseInt(index));
        }
        catch(MyException e){
            System.out.println(e.getMessage());
        }
    }
}

File: map/Interpreter2.0/interface/AllStepsCommand.cs
using System;
using Controller;
namespace Interface
{
    public class AllStepsCommand : Command
    {
        private Controller.Controller ctr;
        public AllStepsCommand(String key, String desc,Controller.Controller ctr) : base(key, desc){
            this.ctr=ctr;
        }
        override public void execute() {
   
            System.Console.WriteLine("Index of Program :");
            String index = System.Console.ReadLine();
            try {
                ctr.AllSteps(Int32.Parse(index));
            }
            catch(MyException.MyException e){
                System.Console.WriteLine(e.getMessage());
            } 
        }

    }
}